vernac/g_vernac.mlg

```diff
@@ -490,6 +490,10 @@ let warn_deprecated_include_type =
   CWarnings.create ~name:"deprecated-include-type" ~category:"deprecated"
          (fun () -> strbrk "Include Type is deprecated; use Include instead")
 
+let warn_deprecated_bind_several_scopes =
+  CWarnings.create ~name:"deprecated-bind-multiple-scopes" ~category:"deprecated"
+         (fun () -> strbrk "Binding of several scopes at the same time is deprecated. ")
```

Comment:
This message is a bit confusing. I thought the scope was e.g. `Z_scope`. If so, then the command always take one single scope but binds several items to this scope at once.

Reply:
I tried another formulation: "Binding a scope to several types at the same time is deprecated".

Reply:
Sounds better.